ocaml/idl/ocaml_backend/ocaml_syntax.ml

```diff
@@ -77,33 +77,42 @@ end
 
 module Module = struct
   type e = Let of Let.t | Module of t | Type of Type.t
-  and t = { name: string;     (** OCaml module name *)
-            preamble: string list; (** Convenient place for helper functions, opens etc *)
-            letrec: bool;     (** True for all the let bindings to be mutually recursive*)
-            args: string list; (** for functor *)
+  and t = { name: string;           (** OCaml module name *)
+            preamble: string list;  (** Convenient place for helper functions, opens etc *)
+            postamble: string list; (** True for all the let bindings to be mutually recursive*)
+            letrec: bool;           (** for functor *)
+            args: string list;
             elements: e list }
 
-  let make ?(preamble=[]) ?(letrec=false) ?(args=[]) ~name ~elements () =
-    { name = name; preamble = preamble; letrec = letrec;
-      args = args; elements = elements }
+  let make ?(preamble=[]) ?(letrec=false) ?(args=[]) ?(postamble=[]) ~name ~elements () =
+    { name = name;
```

Comment:
Since your touch this: it can be shortened to `{name; preamble; postamble; letrec; args; elements}`.